src/main/java/nextstep/step2/domain/Line.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Line {
+    private Random random = new Random();
+    private List<Boolean> points = new ArrayList<>();
+
+    public Line (int userCount) {
+        IntStream.range(0, userCount-1)
+                .mapToObj(index -> points.add(isAbleDrawLine(index)))
+                .collect(Collectors.toList());
```

Comment:
GenerateLadder에서 하신 것처럼
```suggestion
    private List<Boolean> points;

    public Line (int userCount) {
        points = IntStream.range(0, userCount-1)
                .mapToObj(index -> isAbleDrawLine(index))
                .collect(Collectors.toList());
```
로 바꾸셔도 좋겠습니다:)

Reply:
isAbleDrawLine() 에서 이전 줄이 있는지 없는지 여부를 판단하는 로직이 있습니다 (point.get(idx-1))
그래서 순차적으로 point값을 넣어주지 않으면 정상동작하지 않아서 그대로 두었습니다.

해당 로직은 단순 반복문을 통해 구현해도 상관없었을거 같습니다.